services/text-formatters.spec.js

```diff
@@ -60,6 +60,27 @@ describe('Text formatters', function () {
     given(1100000000000000000000).expect('1.1Z')
     given(2222222222222222222222222).expect('2.2Y')
     given(22222222222222222222222222).expect('22Y')
+    given(-999).expect('-999')
```

Comment:
lets add a test for zero in here too

```suggestion
    given(22222222222222222222222222).expect('22Y')
    given(0).expect('0')
    given(-999).expect('-999')
```